apps/studio/src/features/editing-experience/components/CreateCollectionPageModal/DetailsScreen.tsx

```diff
@@ -179,34 +184,39 @@ export const CreateCollectionPageDetailsScreen = () => {
                     URL should be short and simple
                   </FormHelperText>
                 </FormLabel>
-                <InputGroup>
```

Comment:
Can we keep this? I thought it made it more intuitive for new users on Studio

Reply:
see above